### ISSUE TITLE
Expose PHP in WordPress

### ISSUE THREAD

**Issue by ComputerCarl** [caprover/one-click-apps#68]
Modifying the WordPress upload size is a very common use-case, as discussed in issue [#175](https://github.com/caprover/caprover/issues/175). I would like to suggest adding a second persistent volume;
```
                "volumes": [
                    "$$cap_appname-wp-data:/var/www/html",
                    "$$cap_appname-php:/usr/local/etc/php"
                ],
```

**Comment by githubsaturn** [comment]
You can always add this volume post installation. What we try to stay away from is that there is a lot of common use cases, if we incorprate all of them, then the installation process becomes for complex for newbies. 

On the other hand, experienced developers always know what they need to change and they can go ahead and modify whatever value they need to. As a general rule of thumb, as long as, the modification is possible, we don't add it to the general flow in order to keep the general flow smoother.

**Comment by ComputerCarl** [comment]
I agree that practically anything can be done post-installation. Perhaps allowing creation of persistent custom templates or the ability to use a standard docker-compose would alleviate the need to modify pre-defined templates.

**Comment by derek-adair** [comment]
Off topic... genuine question...

 but isn't this a bad idea for security reasons?

**Comment by rafaelstz** [comment]
To customize it and keep using the native Docker from WordPress you just need to access your server, create a file uploads.ini which is including your changes, and set this file as persistent, I created a file in this path below with the respective content.

    /var/lib/docker/volumes/caprover--rafael-wp-data/uploads.ini

```
file_uploads = On
memory_limit = 500M
upload_max_filesize = 500M
post_max_size = 500M
max_execution_time = 600
```

After setting it, your container will be reloaded and it's going to be like this image below.

![Screen Shot 2021-04-10 at 12 31 09 AM](https://user-images.githubusercontent.com/610598/114258374-15824300-9994-11eb-84c6-df3cd152ad23.png)
